### DISCUSSION TITLE
Will this extension work on other platforms like Safari, Firefox, Opera, etc.?

### DISCUSSION THREAD

**Discussion by joaocanuto** [ErnaneJ/tradingworks-plus#24] [Q&A] [answered]
As a Safari user, I'm thrilled to incorporate this extension into my daily work routine. Are there any plans for your development team to extend support to other web browsers?
Btw, Thank you for sharing this with us. It's greatly appreciated and very helpful.

**Comment by ErnaneJ** [comment, marked as answer]
Unfortunately this is currently not something urgent on our to-do list. The development of the extension is being done directly based on chrome, which is perpetuated in any browser based on chromiun, which is not the case with Safari, for example.

On the other hand, keep an eye on the repository because although it is not one of our highest priorities, it is still something present in our agenda and that should be implemented in the future.
